data/json/recipes/weapon/ranged.json

```diff
@@ -17,7 +17,68 @@
     "components": [
       [ ["nailgun", 1] ],
       [ ["pipe", 1] ],
-      [ ["2x4", 2], [ "stick", 3 ] ]
+      [
```

Comment:
Well you have mixed styles here anyway. It's clearer to put all the alternatives on a single line